mysite/wsgi.py

```diff
@@ -6,4 +6,4 @@
 
 os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'mysite.settings')
 
-application = get_wsgi_application()
+application = get_wsgi_application()  # pylint: disable=invalid-name
```

Comment:
This doesn't classify as

> Add pylint good-names

You should probably change it to smth like "Set up pylint exceptions"